03_SageMakerPipelinesAndDeploy/main.R

```diff
@@ -0,0 +1,87 @@
+# Make sure you have all the necessary packages installed ------------------
```

Comment:
This file should be written as Rmd like the other modules in this. Can combine it with the README.md